change/@fluentui-react-f596f7cc-faeb-404e-9c21-006e9137848e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Since this has the potential to impact behavior for teams that are using `text: ''` as an alternative to `null`/`undefined`, could you update this to be `minor`?

I think it's a really small possibility, but it's probably better to be safe.

Reply:
Good point :)

Amended.